react-transition-group/build.boot

```diff
@@ -1,13 +1,13 @@
 (set-env!
-  :resource-paths #{"resources"}
```

Comment:
Minor note about indentation here, most of your PRs change indentation of at least `set-env!` call. I prefer indenting macro forms by two spaces, similar to functions: https://github.com/bbatsov/clojure-style-guide/pull/153 Probably your editor is currently configured to automatically change indentation for these forms.

Use of either style here is okay, but keeping the existing indentation of the file keeps the diff cleanest which helps with reviews a bit.

Reply:
Yeah it's applying a style I use elsewhere -- I'll be more conscientious of it in future PRs.